### ISSUE TITLE
Where are the tests

### ISSUE THREAD

**Issue by mle-enso** [chrgue/give-a-gif#4]
I can find only one test class and that does nothing.
Please consider adding tests especially to the
```java
@GetMapping("/")
```
endpoint of the controller including its correct output rendering, to facilitate others to contribute.

**Comment by chrgue** [comment]
We now have tests for the GetMapping and for the InMemoryPostRepository.
I guess this issue can be closed?

**Comment by mle-enso** [comment]
No, this issue has to stay open especially until the controller tests have meaningful body assertions, which guarantee that no template variables are left in the rendered response.
